README.md

```diff
@@ -928,91 +935,264 @@ Backup specifies details about data backup procedures.
+Quality attributes are checks that can be applied to the data to ensure its quality. Data can be verified by executing these checks through a data quality engine.
 
+Quality attributes can be:
+- Text: A human-readable text that describes the quality of the data.
+- SQL: An individual SQL query that returns a single value that can be compared.
+- Predefined Types: Some commonly-used predefined quality attributes such as `row_count`, `unique`, `freshness`
+- Vendor-specific: Quality attributes that are specific to a vendor, such as Great Expectations, SodaCL or Montecarlo.
 
-#### SodaCL Quality Object
+A quality object can be specified on field level, or on model level. The top-level quality object are deprecated.
 
-Quality attributes in [Soda Checks Language](https://docs.soda.io/soda-cl/soda-cl-overview.html).
+The fields of the quality object depends on the quality `type`.
 
-The `specification` represents the content of a `checks.yml` file.
+#### Text
+
+ Applicable on: [ ] top-level, [x] model, [x] field
+
+A human-readable text that describe the quality of the data. Later in the development process, these might be translated into an executable check (such as `sql`), or checked through an AI engine.
 
-Example (inline):
+| Field       | Type     | Description                                                        |
+|-------------|----------|--------------------------------------------------------------------|
+| type        | `string` | `text`                                                             |
+| name        | `string` | Optional. A human-readable name for this check                     |
+| description | `string` | A plain text describing the quality attribute in natural language. |
+
+Example:
 
 ```yaml
-quality:
-  type: SodaCL   # data quality check format: SodaCL, montecarlo, dbt-tests, custom
-  specification: # expressed as string or inline yaml or via "$ref: checks.yaml"
-    checks for orders:
-      - row_count > 0
-      - duplicate_count(order_id) = 0
-    checks for line_items:
-      - row_count > 0
+models:
+  my_table:
+    fields:
+      email:
+        quality:
+          - type: text
+            description: The email address was verified by the system
 ```
 
-Example (string):
+#### SQL
+
+Applicable on: [ ] top-level, [x] model, [x] field
+
+An individual SQL query that returns a single number or boolean value that can be compared. The SQL query must be in the SQL dialect of the provided server.
+
+| Field                            | Type                   | Description                                                |
+|----------------------------------|------------------------|------------------------------------------------------------|
+| type                             | `string`               | `sql`                                                      |
+| query                            | `string`               | A SQL query that returns a single number or boolean value. |
```

Comment:
empty set for true would also be nice

Reply:
let's stick with soda's threshold values.